website/src/components/example/styled.ts

```diff
@@ -33,6 +33,10 @@ export const ExampleCard = styled.a`
     transition-property: filter;
     transition-duration: var(--ifm-transition-slow);
     transition-timing-function: var(--ifm-transition-timing-default);
+    height: 100%;
+    height: -moz-available;
+    height: -webkit-fill-available;
+    height: fill-available;
```

Comment:
Just making sure that this does not stretch the image? It is important that we preserve the aspect ratio.

Reply:
Should we then set the images as a background to prevent future cases?

Reply:
Perhaps. You could also check how this is done in deck.gl examples page. You can inspect styles via the Chrome debug window.

Reply:
Deck.gl has same sized images in the gallery so it's not the case.
I can ask our designers to prepare good quality images in one size also

Reply:
Sound good. let's cut the images to size.